Makefile

```diff
@@ -33,12 +34,22 @@ build:
 
 
 .PHONY: docker
-docker: build
-	mvn install -f docker/pom.xml -Ddocker.repo.name=${DOCKER_REPO} -Ddocker.image.tag=${DOCKER_IMAGE_TAG}
+docker: 
+	[ -d "docker/global-apim/target" ] || mvn initialize -f docker/pom.xml
+	cd docker/global-apim; \
+	docker build -t ${DOCKER_REPO}/wso2am:${DOCKER_IMAGE_TAG} .
+	cd docker/microgateway/init-container; \
```

Comment:
Is this path correct?

Reply:
Yes, it is the relative path from the project root directory